lib/widget/dropdown.dart

```diff
@@ -210,14 +238,25 @@ class SelectOneState extends FormFieldWidgetState<SelectOne> {
     if (items != null) {
       results = [];
       for (SelectOneItem item in items) {
-        item.icon != null
-            ? results.add(
+        item.isIcon != null || item.isIcon == true
+            ?
+        results.add(
           DropdownMenuItem(
             child: Row(
               children: [
-                iconframework.Icon.fromModel(item.icon!),
-                const SizedBox(
-                  width: 10.0,
+                item.icon != null
+                    ? iconframework.Icon.fromModel(item.icon!)
+                    : SizedBox(
+                  width: widget._controller.size.toDouble(),
+                ),
+                SizedBox(
+                  width: item.icon != null
+                      ? item.icon!.size != null
+                      ? item.icon!.size == widget._controller.size
+                      ? 10
+                      : checkDifference(item.icon!.size!)
+                      : widget._controller.size.toDouble() + 10.0
```

Comment:
Is this correct? The icon still have a size even if the size is not set right?

Reply:
yes this logic is correct

Reply:
I can added sizedbox here and sized box width is 10 because is icon added than add space between icon and text that's why i added 10.0 space between us